Dockerfile

```diff
@@ -16,12 +16,14 @@ ARG BASE_IMAGE=nvcr.io/nvidia/tritonserver:20.11-py3
 ARG TRITONSDK_BASE_IMAGE=nvcr.io/nvidia/tritonserver:20.11-py3-clientsdk
 
 ARG MODEL_ANALYZER_VERSION=1.0.0dev
+ARG MODEL_ANALYZER_WHEEL_VERSION=1.0.0.dev0
```

Comment:
No, we cannot do this. Why does the old one above not work, when we can get it to work in all other cases?  I absolutely do not want to VERSION strings.

Reply:
pip takes `1.0.0dev` and turns it into `1.0.0.dev0`. I added this only so that we can use this arg in line 55.